tools/generate_support_bundle.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
This should use the env shebang as well 
`#!/usr/bin/env python3`
instead of hardcoded python path

Reply:
Fixed